frontend/src/components/Collections/components/Grid/components/Row/CollectionRow/index.tsx

```diff
@@ -75,13 +76,14 @@ const CollectionRow: FC<Props> = (props) => {
   } = aggregateDatasetsMetadata(datasets);
 
   return (
-    <StyledRow>
+    <StyledCollectionRow
+      onClick={(e) => {
+        if ((e.target as Element).tagName !== "A")
+          navigate(`/collections/${id}${isPrivate ? "/private" : ""}`);
+      }}
+    >
```

Comment:
Entire row will now navigate to the collection page, ignore row clicks if the user is clicking on the DOI link